File: Java/src/Faccat/Exercicio10.java
package Faccat;

import java.util.Scanner;

public class Exercicio10 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite o valor de fábrica: ");
        double fabrica = scanner.nextDouble();

        System.out.print("Digite o percentual do distribuidor: ");
        double distribuicao = scanner.nextDouble();

        System.out.print("Digite o percentual dos impostos: ");
        double imposto = scanner.nextDouble();

        double valor = fabrica + (fabrica * distribuicao) / 100 + (fabrica * imposto) / 100;

        System.out.println("O valor final do carro é: R$" + valor);

        scanner.close();
    }
}

File: JavaScript/Faccat/Exercicio10.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

rl.question('Digite o valor de fábrica: ', (fabrica) => {
  rl.question('Digite o percentual do distribuidor: ', (distribuicao) => {
    rl.question('Digite o percentual dos impostos: ', (imposto) => {
      fabrica = parseFloat(fabrica);
      distribuicao = parseFloat(distribuicao);
      imposto = parseFloat(imposto);

      const valor = fabrica + (fabrica * distribuicao) / 100 + (fabrica * imposto) / 100;
      console.log(`O valor final do carro é: R$ ${valor.toFixed(2)}`);

      rl.close();
    });
  });
});






